test/ergw_http_api_SUITE.erl

```diff
@@ -95,7 +95,7 @@ all() ->
 init_per_suite(Config0) ->
     inets:start(),
     Config1 = [{app_cfg, ?TEST_CONFIG},
-              {handler_under_test, ggsn_gn_proxy}
+              {handlers_under_test, [ggsn_gn_proxy]}
```

Comment:
pure whitespace cleanup changes are ok, but only if you don't mix them with other changes. Make a single cset that collects all white space cleanups if needed.